modules/auxiliary/scanner/http/cisco_ironport_enum.rb

```diff
@@ -60,10 +60,14 @@ def check_conn?
         'uri'       => '/',
         'method'    => 'GET'
       })
-      print_good("#{rhost}:#{rport} - Server is responsive...")
-      return 1
+      if res
```

Comment:
I'm not sure what the appropriate response code should be. Probably 200. I'm not sure it matters, since the check is already simplistic.